src/Extensions/Compatability/Validation/ValidatedNotNullAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.Omex.Extensions.Compatability.Validation;
+
+namespace Microsoft.Omex.System.Validation
+{
+	/// <summary>
+	/// Indicates to code analysis tools that a method validates a particular parameter to not be null.
+	/// 
```

Comment:
>  [](start = 4, length = 1)

Nitpick: trailing whitespace on a number of these lines

Reply:
Fixed